src/lib/components/Instances/InstanceTable.svelte

```diff
@@ -130,10 +134,20 @@
 					<Button classNames="text-red-600" onclick={() => (showDelete = true)}>
 						<Trash2 class="h-5 w-5" />
 					</Button>
-					<GenerateInstances {createInstances} similar={true} />
+					<GenerateInstances
+						{createInstances}
+						similar={true}
+						showDataGeneration={false}
+						dataGenerationInstruction={''}
+					/>
 				</div>
 			{/if}
-			<GenerateInstances {createInstances} similar={false} />
+			<GenerateInstances
+				{createInstances}
+				similar={false}
+				bind:showDataGeneration
+				bind:dataGenerationInstruction
+			/>
```

Comment:
Wouldn't it make sense to switch the button instead of having both? Kinda confusing maybe.